app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -76,11 +100,39 @@ def get_all_records():
     # If request was not successful, there will be no records field in response
     # Just return what is in cached layer and log an error
     except KeyError as e:
-        body = "Results were not successfully retrieved from Airtable API. " \
-               "Please check connection parameters in config.py and fields in airtable_fields_config.json."
-        request_info = {
-            "url": url,
-            "headers": json.dumps(headers)
-        }
-        send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
-        return request_info
+        _handle_airtable_error(e, data, url)
+
+'''
+Updates population_group_options table in whiteclaw with entries from Sample Frame GOI table in Serosurveillance Base in Airtable (name, french translation and order).
+These records will be used for the "Population group" filters on serotracker.com.
+'''
+def ingest_sample_frame_goi_filter_options():
+    data = _airtable_get_request(AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL)
```

Comment:
can we kill the naming convention of "_function name" for private functions? it was a pattern that I probably started but I don't think we use it consistently anymore so let's just keep everything formatted like "function_name" instead of "_function_name"

Reply:
cleaned it up for this file!